config.yaml

```
@@ -62,6 +62,7 @@ episodes:
 - 2-documentation.md
 - 3-testing.md
 - 4-ci.md
+- 5-modular-code.md
```

```suggestion
- 1-modular-code.md
- 2-documentation.md
- 3-testing.md
- 4-ci.md
```
I think this is usually the order that we teach it right? Can you also change the filename to `episodes/1-modular-code.md` accordingly?